### DISCUSSION TITLE
Is there a plan to support medical 3D detection tasks?

### DISCUSSION THREAD

**Discussion by ghost** [Project-MONAI/MONAI#1741] [Ideas]
Hi, I'm using MONAI to do detection tasks, I implemented 3D RPN, RetinaNet, etc., but still have some problems such as 3D ROI align, too slow on NMS. Is there any plan to support medical 3D detection tasks?

**Comment by Nic-Ma** [comment]
Hi @tangzhe0220 ,

Thanks for raising this topic.
MONAI working group is already discussing how to support detection tasks, we don't get a standard label format yet.
I think we will try to add the support soon.

Thanks.
